docs/conceptual/array.contains['t]-function-[fsharp].md

```diff
@@ -0,0 +1,75 @@
+---
+title: Array.contains<'T> Function (F#)
+description: Array.contains<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: liboz
+---
+
+# Array.contains<'T> Function (F#)
+
+Evaluates to `true` if the given element is in the given array.
```

Comment:
I expect we could do with a rework of that throughout the documentation ☺

From: Libo Zeng [mailto:notifications@github.com]
Sent: Saturday, June 25, 2016 4:39 AM
To: Microsoft/visualfsharpdocs <visualfsharpdocs@noreply.github.com>
Cc: Kevin Ransom <Kevin.Ransom@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Microsoft/visualfsharpdocs] List.contains and Array.contains (#187)


In docs/conceptual/array.contains['t]-function-[fsharp].md<https://github.com/Microsoft/visualfsharpdocs/pull/187#discussion_r68487828>:

> @@ -0,0 +1,75 @@

> +---

> +title: Array.contains<'T> Function (F#)

> +description: Array.contains<'T> Function (F#)

> +keywords: visual f#, f#, functional programming

> +author: liboz

> +---

> +

> +# Array.contains<'T> Function (F#)

> +

> +Evaluates to `true` if the given element is in the given array.

I tried to follow the wording of the documentation of the Set.contains function, which is: Evaluates to **true** if the given element is in the given set. However, perhaps the wording from the Map.containskey function be more appropriate: Tests if an element is in the domain of the map.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/visualfsharpdocs/pull/187/files/8f591c92a19d68d25b39625fe7b1eceeb09d149c#r68487828>, or mute the thread<https://github.com/notifications/unsubscribe/AE76FkULtW-tlNmPD2Qzcr-p7X-i53xuks5qPRNBgaJpZM4I-PnL>.